workspaces/liferay-sample-workspace/client-extensions/liferay-sample-custom-element-3/src/app/app.component.ts

```diff
@@ -3,13 +3,23 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
-import {Component, Input} from '@angular/core';
+import {Component, Input, OnInit} from '@angular/core';
 
 @Component({
 	selector: 'liferay-sample-custom-element-3',
 	templateUrl: './app.component.html',
 	styleUrls: ['./app.component.css'],
 })
-export class AppComponent {
+export class AppComponent implements OnInit {
 	@Input('title') title = 'liferay-sample-custom-element-3';
+
+	logMessage() {
+
+		// eslint-disable-next-line no-console
+		console.log('liferay-sample-custom-element-3 loaded');
```

Comment:
Not sure if we want this kind of things in examples. I get you are using it for the test but... :thinking:  

@dsanz Any opinions on this?